src/ethereum/tangerine_whistle/spec.py

```diff
@@ -735,9 +735,9 @@ def calculate_block_difficulty(
 
     Parameters
     ----------
-    parent_block_number :
-        Block number of the parent block.
```

Comment:
Why did you remove "parent" from the description? We are passing the parent's block number here.
https://github.com/ethereum/execution-specs/blob/8d24c57914258947466b47a842228d3bf9b648a0/src/ethereum/tangerine_whistle/spec.py#L184-L190

Reply:
Essentially, in frontier, we were passing the `block_number`. For the later forks, we were passing the `parent block number` and then adding 1 to it inside the function. I just wanted to standardise this function to whatever extent possible. I have also updated the corresponding line where this function is eventually called.

Reply:
He changed the semantics as well as the name, but seems to have forgotten to change the callers. It only matters for blocks where `block_number % 100000 == 0` so the test suite hasn't picked it up.

Reply:
Ok, so as peter mentioned you need to fix the callers.

Reply:
Hmm. This is strange cause I distinctly remember updating these callers. Perhaps, I failed to commit some changes.

@voith @petertdavies Thanks for catching this error. I'll update

Reply:
Updated!